src/composables/useFlawModel.ts

```diff
@@ -59,16 +60,21 @@ export function useFlawModel(forFlaw: ZodFlawType = blankFlaw(), onSaveSuccess:
       Object.entries(validatedFlaw.data).filter(([, value]) => value !== '')
     );
 
-    postFlaw(flawForPost)
+    await postFlaw(flawForPost)
       .then(createSuccessHandler({ title: 'Success!', body: 'Flaw created' }))
       .then((response: any) => {
         router.push({
           name: 'flaw-details',
           params: { id: response?.cve_id || response?.uuid },
         });
+        flaw.value.uuid = response.uuid;
       })
       .catch(createCatchHandler('Error creating Flaw'))
       .finally(() => { isSaving.value = false; });
```

Comment:
@superbuggy Can we move this after saveCvssScore? Otherwise LGTM

Reply:
We cannot, this value needs to be set beforehand, and `response` would be out of scope.

Reply:
No problem

Reply:
actually I tried to select line 71. my mistake 😅